code/lib/core-server/src/presets/common-preset.ts

```diff
@@ -196,7 +196,7 @@ export const features = async (
 });
 
 export const csfIndexer: Indexer = {
-  test: /\.(stories|story)\.(m?js|ts)x?$/,
+  test: /(stories|story)\.(m?js|ts)x?$/,
```

Comment:
we use `.examples.tsx` naming convention in our project. Can we somehow override this regexp in our config?

Reply:
No. There's been talk about using the stories glob directly, but that's not implemented. Easiest thing is to use .stories.x per the documentation

Reply:
You could also add a custom indexer yourself. @JReinhold  and @kylegach are working on docs for that

Reply:
Did we get documentation on this?

Reply:
yes @antrianis! The docs won't be published until the next minor version, but you can see it here: https://github.com/storybookjs/storybook/blob/next/docs/api/main-config-indexers.md